api/envoy/config/core/v3/protocol.proto

```diff
@@ -113,6 +113,10 @@ message HttpProtocolOptions {
   // If this setting is not specified, the value defaults to ALLOW.
   // Note: upstream responses are not affected by this setting.
   HeadersWithUnderscoresAction headers_with_underscores_action = 5;
+
+  // Optional maximum requests for a downstream connection.
+  // If not specified, there is no limit.
+  uint32 max_requests_connection = 6;
```

Comment:
Please retain the previous name: `max_requests_per_connection`

Reply:
Sure, I will do that.